### ISSUE TITLE
swww doesnt work on the latest version

### ISSUE THREAD

**Issue by Felix-Kyun** [Horus645/swww#140]
error being 
```
Error: "failed to read serialized length: Resource temporarily unavailable (os error 11)"
```

it does sometime work, and then suddenly doesnt work.
most of the times it works 2-3 times initially when the machine boots.
it worked fine on older version, just updated to `0.8.0` and this problem started occuring.

**Comment by Horus645** [comment]
Ah, I just published a new minor version with many bug fixes. This is probably solved in `0.8.1`.

In case it isn`t, can you please provide more details, as in, what command did you run (just `swww init`, or `swww img`)?

**Comment by Felix-Kyun** [comment]
ok fixed,  it works fine on version `0.8.1`
